views/includes/head.html

```diff
@@ -1,43 +1,9 @@
 <!-- Google Tag Manager -->
-{% if environment === "prod" or environment === "production"%}
-    <!-- Live analytics -->
-    <script>
-        (function (w, d, s, l, i) {
-            w[l] = w[l] || [];
-            w[l].push({
-              'gtm.start': new Date().getTime(),
-              event: 'gtm.js'
-            });
-            var f = d.getElementsByTagName(s)[0],
-              j = d.createElement(s),
-              dl = l != 'dataLayer' ? '&l=' + l : '';
-            j.async = true;
-            j.src =
-              'https://www.googletagmanager.com/gtm.js?id=' + i + dl +
-              '&gtm_auth=s9D_6Cs9pP0A0pPraIxTSA&gtm_preview=env-2&gtm_cookies_win=x';
-            f.parentNode.insertBefore(j, f);
-        })(window, document, 'script', 'dataLayer', 'GTM-N4FNRXM');
-    </script>
-{% else %}
-    <!-- Dev analytics -->
-    <script>
-        (function (w, d, s, l, i) {
-            w[l] = w[l] || [];
-            w[l].push({
-              'gtm.start': new Date().getTime(),
-              event: 'gtm.js'
-            });
-            var f = d.getElementsByTagName(s)[0],
-              j = d.createElement(s),
-              dl = l != 'dataLayer' ? '&l=' + l : '';
-            j.async = true;
-            j.src =
-              'https://www.googletagmanager.com/gtm.js?id=' + i + dl +
-              '&gtm_auth=vVL1r5F0rqfg4e_0meyy0w&gtm_preview=env-31&gtm_cookies_win=x';
-            f.parentNode.insertBefore(j, f);
-        })(window, document, 'script', 'dataLayer', 'GTM-N4FNRXM');
-    </script>
-{% endif %}
+<script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
```

Comment:
Can you please formate the script code

Reply:
it's copy and paste direct from what google provides.  Formatting will make it harder to check if its changed in future against what google provides?